WhatBackend.TelegramBotApi/Startup.cs

```diff
@@ -39,14 +37,15 @@ public void ConfigureServices(IServiceCollection services)
                   b => b.MigrationsAssembly("CharlieBackend.TelegramBotApi"));
             });
 
+            CompositionRoot.InjectDependencies(services, Configuration);
 
+            services.AddScoped<StartCommand>();
             services.AddScoped<HelloCommand>();
             services.AddScoped<GetMarkCommand>();
 
-            services.AddScoped<ICurrentUserService, CurrentUserService>();
-            services.AddScoped<IUnitOfWork, UnitOfWork>();
-            services.AddScoped<IDashboardRepository, DashboardRepository>();
-            services.AddScoped<IDashboardService, DashboardService>();
+            services.AddEasyNetQ(Configuration.GetConnectionString("RabbitMQ"));
```

Comment:
Is there any purpose in adding RabbitMQ and AzureBlobs in bot project? Consider using only needed dependencies.